deployments/local-external-minio/README.md

```diff
@@ -0,0 +1,31 @@
+# Local External Minio Deployment
```

Comment:
:pencil2: It was not immediately clear why there were two local deployments. Claify in the readme. Something like "The local-external-minio should be used if a minio services is already running on port 9000."

Reply:
The readmes already include those details. Is this not sufficient?
```markdown
# Local External Minio Deployment

## Description
This is an example local deployment of the Ingest Service that connects to a local Minio server and
uses that for the S3 store. It also includes a deploy.sh script that will attempt to create the
docker network needed.

## Prerequisites
* Java 11
* Docker daemon
* Linux/Mac
* Minio at http://localhost:9000 with a "metacard-quarantine" bucket and login information defined
in `secrets/minio_access.notsec` and `secrets/minio_secret.notsec`.

...

## Additional Information
Any additional information about the services started can be found in the README.md at the root of
the project.
```
```markdown
# Local Minio Deployment

## Description
This is an example local deployment of the Ingest Service that also starts up a local Minio server
on port 9000, creates an metacard-quarantine bucket, and uses that for the S3 store. It also
includes a deploy.sh script that will attempt to create the docker network needed.

## Prerequisites
* Java 11
* Docker daemon
* Linux/Mac

...

## Minio
The Minio web client is exposed on port 9000 with the default login information:
Access Key: MINIOEXAMPLEACCESSKEY
Secret key: MINIOEXAMPLESECRETKEY
These keys can also be found in `secrets/minio_access.notsec` and `secrets/minio_secret.notsec`.

## Additional Information
Any additional information about the services started can be found in the README.md at the root of the project.
```

Reply:
I didn't think so.

Reply:
:+1: I'm going to keep it as it for now. Feel free to improve these readmes after this is merged.